entities/entities/rvr_storage/init.lua

```diff
@@ -0,0 +1,43 @@
+AddCSLuaFile( "cl_init.lua" )
+AddCSLuaFile( "shared.lua" )
+
+include( "shared.lua" )
+
+function ENT:Initialize()
+    -- Default model, expected to be changed by derived classes
+    self:SetModel( "models/props_junk/wood_crate001a.mdl" )
```

Comment:
would the derived class have to override intialize. perhaps this should be stored in the ENT table `ENT.model`.
same with line 24 should that structure be in its own variable or function.


although a derived class could just call the base classe's initialize and then update those values again